### ISSUE TITLE
What is the intended way to move data from tingodb to mongodb?

### ISSUE THREAD

**Issue by shedar** [sergeyksv/tingodb#92]
As mentioned on tingodb site, "Risk free because switching to MongoDB is seamless and will not need code changes."
But I can't find any manuals, how I can move my data from tingodb to mongodb.

Is there are any better way then writing a custom "select from tingo - insert to mongo" tool?

**Comment by sergeyksv** [comment]
No, only select - insert is possible, though in schema less word this is just couple of code line. I'll keep this request open. We though about something like making tingodump, tingorestore to work with dumps compatible with mongodb.

**Comment by kgividen** [comment]
+1

**Comment by yangyaochia** [comment]
+1
